system/include/libcxx/atomic

```diff
@@ -539,7 +539,8 @@ void atomic_signal_fence(memory_order m) noexcept;
 
 _LIBCPP_BEGIN_NAMESPACE_STD
 
-#if !__has_feature(cxx_atomic) && _GNUC_VER < 407
+// #if !__has_feature(cxx_atomic) && _GNUC_VER < 407 // XXX Emscripten: Added ' && !defined(__EMSCRIPTEN__)'
+#if !__has_feature(cxx_atomic) && _GNUC_VER < 407 && !defined(__EMSCRIPTEN__)
```

Comment:
@kripken : Do you know if our codebase somehow implements a built-in `__has_feature`? We would like to always advertise `__has_feature(cxx_atomic)` as true, but could not find where that would exist. All I found was `#define __has_feature(x) 0` in our headers.